### ISSUE TITLE
Issues with angular2-jwt v1.0

### ISSUE THREAD

**Issue by Smiley01987** [mgechev/angular-seed#2079]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see use [gitter](https://gitter.im/mgechev/angular2-seed) or [stackoverflow](https://stackoverflow.com/questions/tagged/angular2)
```

**Current behavior**
Hi, i just added angular2-jwt v1.0 as a dependency (https://github.com/auth0/angular2-jwt/tree/v1.0), but when i run yarn run build.prod.aot i get the following error;

> [12:08:38] 'build.bundles.app' errored after 746 ms
[12:08:38] Error on dependency parsing for dist/tmp/app/app.module.js at file:///xxx/xxx/xxx/xxx/xxx/xxx/project/dist/tmp/app/app.module.js
Loading dist/tmp/app/main.js
(SystemJS) ENOENT: no such file or directory, open '/xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json'
Error: ENOENT: no such file or directory, open '/xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json'
at Error (native)
Error loading /xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json
[12:08:38] 'build.prod' errored after 11 s
[12:08:38] Error in plugin 'run-sequence(build.bundles.app)'
Message:
Error on dependency parsing for dist/tmp/app/app.module.js at file:///xxx/xxx/xxx/xxx/xxx/xxx/project/dist/tmp/app/app.module.js
Loading dist/tmp/app/main.js
(SystemJS) ENOENT: no such file or directory, open '/xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json'
Error: ENOENT: no such file or directory, open '/xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json'
at Error (native)
Error loading /xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json
Details:
originalErr: Error: ENOENT: no such file or directory, open '/xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json'
Stack:
Error on dependency parsing for dist/tmp/app/app.module.js at file:///xxx/xxx/xxx/xxx/xxx/xxx/project/dist/tmp/app/app.module.js
Loading dist/tmp/app/main.js
(SystemJS) ENOENT: no such file or directory, open '/xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json'
Error: ENOENT: no such file or directory, open '/xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json'
at Error (native)
Error loading /xxx/xxx/xxx/xxx/xxx/xxx/project/node_modules/@auth0/package.json
error Command failed with exit code 1.

It looks like it's pointing to the wrong `package.json` file (should point to `node_modules/@auth0/angular-jwt`). 

**Expected behavior**
Being able to use angular2-jwt v1.0

**Please tell us about your environment:**

* **Angular Version:** `4.3.6`
* **Node:** v6.7.0
* **Package manager:** Yarn
* **OS:** macOS Sierra 10.12.6 (16G29)

**Comment by fisenkodv** [comment]
@Smiley01987, have you added
```
      {
        name: 'angular2-jwt',
        path: 'node_modules/angular2-jwt/angular2-jwt.js'
      }
```

into your project.config.ts?

**Comment by sparachi** [comment]
similar issue with auth0-js package. the comment [here](https://github.com/mgechev/angular-seed/issues/2176#issuecomment-361243964) helped. Added format: global 

let additionalPackages: ExtendPackages[] = [
      {
        name: 'auth0-js',
        path: 'node_modules/auth0-js',
        packageMeta: {
          format: 'global',
          main: 'build/auth0.js',
          defaultExtension: 'js'
        }
      }]

this.addPackagesBundles(additionalPackages);

the error I got was Error: Multiple anonymous defines. 
build.bundles.app Error on dependency parsing for auth0-js
